code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -772,6 +772,12 @@ datum
 			result = "plantbgone"
 			required_reagents = list("toxin" = 1, "water" = 4)
 			result_amount = 5
+		luminol
+			name = "Luminol"
+			id = "luminol"
+			result = "luminol"
+			required_reagents = list("hydrogen" = 2, "carbon" = 2, "ammonia" = 2)

```

Comment:
This formula is really weird. Why don't you just make it 1+1+1 = 3 instead of 2+2+2=6?

For example, if someone puts in 25 of each reagent it will only make 72 Luminol and leave 1 of each reagent.